### PR TITLE
clientv3: document client-side request size limit

### PR DESCRIPTION
**Pull request by gyuho**
Separated from https://github.com/coreos/etcd/pull/9129.

### CODE REVIEW

File: clientv3/README.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ The etcd client optionally exposes RPC metrics through [go-grpc-prometheus](http
 
 The [namespace](https://godoc.org/github.com/coreos/etcd/clientv3/namespace) package provides `clientv3` interface wrappers to transparently isolate client requests to a user-defined prefix.
 
+## Request size limit
```
Comment (xiang90) [inline clientv3/README.md]: remove the server side mentioning here. is the recv msg size also configurable?

**Comment by gyuho** [comment]
@xiang90 PTAL. Removed server-mentions.

**Comment by xiang90** [comment]
>with corresponding `--max-request-bytes` etcd server flag.

i still can see this in this pr.

File: clientv3/README.md [hunk 2]
```diff
@@ -80,6 +80,10 @@ The etcd client optionally exposes RPC metrics through [go-grpc-prometheus](http
...
+
+As of v3.2.12, the client size limit is configurable via `clientv3.Config.MaxCallSendMsgSize` and `MaxCallRecvMsgSize` in bytes, with corresponding `--max-request-bytes` etcd server flag. If none given, client request send limit defaults to 2 MiB including gRPC overhead bytes. And receive limit defaults to `math.MaxInt32`.
```
Comment (xiang90) [inline clientv3/README.md]: remove the client version mentioning?

**Comment by xiang90** [comment]
defer to @jpbetz

**Pull request merged**